theme-dark.scss

```
@@ -0,0 +1,8 @@
+/*-- scss:defaults --*/
+$body-bg: #181818;
+$body-color: white;
+$link-color: #75AADB;
+$sidebar-bg: #303030;
+$code-background-alpha: -.8;
+$popover-bg: #242424;
+$input-bg: #242424;
```

Maybe we should change the code style. Right now it is a bit hard to read.

```suggestion
$input-bg: #242424;
$code-block-bg: #303030;
$code-bg: #303030;
$code-color: #b84bb8;

```

---

These are good updates!